File: lol/src/main/kotlin/com/your/lol/dto/riot/SummonerDto.java
package com.your.lol.dto.riot;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class SummonerDto {
    private String id;
    private String accountId;
    private String puuid;
    private String name;
    private String summonerLevel;
}

File: ggtics-recruitement-lol-score/src/main/kotlin/com/your/lol/dto/riot/SummonerDto.kt
package com.your.lol.dto.riot

import lombok.Data
import lombok.NoArgsConstructor

class SummonerDto(val id: String,
                  val accountId: String,
                  val puuid: String,
                  val name: String,
                  val summonerLevel: String) {

    private constructor() : this("", "", "", "", "")
}
